### ISSUE TITLE
Installer crashes with error message

### ISSUE THREAD

**Issue by mok0** [pi-hole/pi-hole#2671]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*.
- [x] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the the source code relevant to this issue?:**

1

---
**Expected behaviour:**

The Pi-hole install script completes without binary data in the gravity.sh script.

**Actual behaviour:**
```
  [✓] Restarting pihole-FTL service...
/opt/pihole/gravity.sh: line 18: $'\240\343\004': command not found
/opt/pihole/gravity.sh: line 18: regexconverso.4libwmf-0.2.so.7libkmod.so.2libwayland-cursor.so.0libogg.so.0libdcerpc-binding.so.0libnetpbm.so.10libgstvideo-1.0.so.0libgnomeui-2.so.0libnl-genl-3.so.200libcups.so.2libgcj-tools.so.17libsamba-util.so.0libbs2b.so.0libxkbcommon-x11.so.0libWildMidi.so.2libgstvideo-0.10.so.0libexif.so.12libnss_compat.so.2libbsd.so.0libxcb-render.so.0libORBitCosNaming-2.so.0libopencv_calib3d.so.2.4libvo-aacenc.so.0liblz4.so.1libresolv.so.2libuuid.so.1libkpathsea.so.6libflite_cmu_time_awb.so.1libssh2.so.1libodfgen-0.1.so.1libpixman-1.so.0libgstrtsp-0.10.so.0libgiblib.so.1libustr-1.0.so.1libcdda_paranoia.so.0libwpd-0.10.so.10libmmal_vc_client.solibmmal_util.solibnss_files.so.2libgeany.so.0libarchive.so.13libv4l2rds.so.0libqmi-glib.so.5libcwidget.so.3libxcb-randr.so.0libavc1394.so.0libmpiseq_seq-4.10.0.solibabw-0.1.so.1libhyphen.so.0libwayland-server.so.0libspectre.so.1libXpm.so.4libpytalloc-util.so.2libfakeroot-0.solibOsiCbc.so.3liboauth.so.0libgstcontroller-1.0.so.0libmwaw-0.3.so.3libhogweed.so.4libGLESv2.so.2libxcb-dri2.so.0libQt5X11Extras.so.5libavahi-common.so.3libXmu.so.6libsamba-passdb.so.0libpanel.so.5libgstinsertbin-1.0.so.0libicuio.so.57libcdio_paranoia.so.1libXss.so.1libpam.so.0libubsan.so.0libpgm-5.2.so.0libmmal_core.solibaspell.so.15libvlccore.so.9libvcos.solibgdk_pixbuf_xlib-2.0.so.0libboost_system.so.1.61.0libboost_system.so.1.62.0libopenmpt.so.0libpolkit-gobject-1.so.0libIlmImfUtil-2_2.so.22libaudiofile.so.1librubberband.so.2libLLVM-3.9.so.1libtwolame.so.0libply.so.4libreadline.so.7libapt-pkg.so.5.0libpython3.5m.so.1.0liborc-test-0.4.so.0libpam_misc.so.0libmmal_components.solibmplex2-2.1.so.0libdns-export.so.162libflite_cmu_us_rms.so.1librevenge-stream-0.0.so.0libuno_cppu.so.3libasan.so.3libcairo.so.2libwrap.so.0libgstcdda-0.10.so.0libxcb-xkb.so.1libshout.so.3libe-book-0.1.so.1libgstgl-1.0.so.0libnghttp2.so.14libmms.so.0libacl.so.1libmpeg2encpp-2.1.so.0libcupsimage.so.2libdatrie.so.1libGeoIP.so.1libgcrypt.so.20libbfd-2.28-system.solibmodplug.so.1libgstnet-0.10.so.0libgstaudio-0.10.so.0libQt5Concurrent.so.5libnsl.so.1libldb.so.1libopenjp2.so.7libxfce4util.so.7libnih-dbus.so.1libImlib2.so.1libSoundTouch.so.1libCbcSolver.so.3libiec61883.so.0libkeyutils.so.1libflite_cmu_indic_lang.so.1libfm.so.4libcanberra-gtk3.so.0libXt.so.6libdvbpsi.so.10libXxf86dga.so.1libraw.so.15libQt5Gui.so.5libxcb-xv.so.0libcompface.so.1libip6tc.so.0libxcb-keysyms.so.1libpipeline.so.1libfakeroot-0.solibrdf.so.0libCgl.so.1libXrandr.so.2libspandsp.so.2libevview3.so.3libthai.so.0libgomp.so.1libEGL.so.1libossp-uuid++.so.16libdns-pkcs11.so.162libofa.so.0libcontainers.solibltdl.so.7libgnome-menu-3.so.0liblber-2.4.so.2libflite_cmu_us_kal.so.1libSDL-1.2.so.0libnssutil3.solibpangocairo-1.0.so.0libsndio.so.6.1libcryptsetup.so.4libporttime.so.0libcdda_interface.so.0libpspell.so.15libboost_iostreams.so.1.60.0libasound.so.2libvdpau.so.1libCbc.so.3libmpcdec.so.6libmpg123.so.0libhttp_parser.so.2.8libxml2.so.2libgstallocators-1.0.so.0libebur128.so.1liblzo2.so.2libfontenc.so.1libatasmart.so.4libgstriff-1.0.so.0libgstplayer-1.0.so.0libply-boot-client.so.4libmm-glib.so.0liblwres.so.141libSegFault.solibmatroska.so.6libsamba-credentials.so.0libk5crypto.so.3libgstrtp-0.10.so.0libdcerpc.so.0libgltf-0.0.so.0libcupsmime.so.1libvpx.so.4libpoppler.so.64libavahi-gobject.so.0libmenuw.so.5libmount.so.1libqwt-qt5.so.6libgudev-1.0.so.0libplc4.solibgck-1.so.0libnss_mdns6.so.2libfreetype.so.6libsecret-1.so.0libopenal.so.1libxcb-sync.so.1libbind9.so.140libncursesw.so.5libdb-5.3.solibrest-0.7.so.0libUsageEnvironment.so.3libx265.so.95libsigc-2.0.so.0libbluray.so.1libssl3.solibcares.so.2libisc-export.so.160libXfont.so.1libsuitesparseconfig.so.4libtevent-util.so.0libhistory.so.6libz.so.1libtalloc.so.2libapt-private.so.0.0libfribidi.so.0libmmal.solibglib-2.0.so.0libv4l1.so.0libusbmuxd.so.4libudisks2.so.0libnfs.so.8libulockmgr.so.1libblkid.so.1libcdio_cdda.so.1libv4lconvert.so.0libSDL2-2.0.so.0libRTIMULib.so.7libQt5EglDeviceIntegration.so.5libdc1394.so.22libportmidi.so.0libyaml-0.so.2libgssapi_krb5.so.2libgeoclue-2.so.0libmtdev.so.1libtinfo.so.5libnih.so.1libgmime-2.6.so.0libc.so.6libsmime3.solibxcb-render-util.so.0liblapack.so.3libcc1.so.0libxcb-glx.so.0libsigc-1.2.so.5libcgroup.so.1libfluidsynth.so.1libenchant.so.1libudev.so.1libtdb.so.1libnpth.so.0libopencv_core.so.2.4libobrender.so.32libgstaudio-1.0.so.0libbrcmEGL.solibOsiClp.so.1libXdamage.so.1libkate.so.1libx264.so.148libpisock.so.9libtiff.so.5libobt.so.2libgstrtp-1.0.so.0libgobject-2.0.so.0liborcus-mso-0.11.so.0libicuuc.so.57libplds4.solibfm-extra.so.4libgstcontroller-0.10.so.0libgnome-2.so.0libtheora.so.0libxcb-util.so.0libmpeg2convert.so.0libpolkit-agent-1.so.0libwnck-1.so.22libpanelw.so.5libvorbisenc.so.2libgdk_pixbuf-2.0.so.0libmpdec.so.2libavahi-client.so.3libhardsid-builder.so.0libldap_r-2.4.so.2libbabeltrace.so.1libdrm_amdgpu.so.1libboost_filesystem.so.1.61.0libplist.so.3libdouble-conversion.so.1libasyncns.so.0libnautilus-extension.so.1libmemusage.solibgailutil.so.18libnss_mdns.so.2libavresample.so.3libjackserver.so.0libpcre.so.3libicudata.so.57libnfnetlink.so.0libthreadutil.so.6libbrcmWFC.solibwiringPi.solibgpgme.so.11libva-wayland.so.1liblouisutdml.so.7libdcerpc-samr.so.0libbrcmOpenVG.solibuno_salhelpergcc3.so.3libflite_cmu_us_awb.so.1libjpegxr.so.0libgstpbutils-1.0.so.0libopcodes-2.28-system.solibjbig2dec.so.0libmumps_common_seq-4.10.0.solibpackagekit-glib2.so.18liblcms2.so.2libvchiq_arm.solibutil.so.1libgstrtsp-1.0.so.0librevenge-0.0.so.0libswscale.so.4libdl.so.2libtasn1.so.6libident.so.0libgstbase-1.0.so.0libcap.so.2libjbig.so.0libgnomevfs-2.so.0libgtk-3.so.0libiculx.so.57libgij.so.17libapt-inst.so.2.0libmagic.so.1libmicrodns.so.0libgio-2.0.so.0libjim.so.0.76libva.so.1libzbar.so.0libupnp.so.6libjxrglue.so.0libp11-kit.so.0libopencv_features2d.so.2.4libQt5OpenGL.so.5libpango-1.0.so.0libraptor2.so.0libsratom-0.so.0libnss_nisplus.so.2libevdev.so.2libzmq.so.5libbonobo-activation.so.4libflite.so.1libssh-gcrypt.so.4libsmbconf.so.0libgd.so.3libXinerama.so.1libopencv_objdetect.so.2.4libsoxr.so.0liblua5.2.so.0libnewt.so.0.52libfaad.so.2libflite_usenglish.so.1libstdc++.so.6libproxy.so.1libgsturidownloader-1.0.so.0liblognorm.so.5libdmumps_seq-4.10.0.solibavahi-glib.so.1libxcb-present.so.0libsmbldap.so.0libcdr-0.1.so.1libxtables.so.12libbabeltrace-dummy.so.1libslang.so.2libatomic.so.1libbrcmGLESv2.solibnettle.so.6libfastjson.so.4libboost_iostreams.so.1.62.0libGLESv1_CM.so.1libuno_purpenvhelpergcc3.so.3libXv.so.1libsord-0.so.0libid3tag.so.0libORBit-2.so.0libnss_dns.so.2libgphoto2.so.6libwebkit2gtk-4.0.so.37libdrm.so.2libfaad_drm.so.2libvte.so.9libQt5XcbQpa.so.5libndr-krb5pac.so.0libpagemaker-0.0.so.0libQt5Network.so.5libdcerpc-server.so.0libpython2.7.so.1.0libGL.so.1libfftw3f_threads.so.3libXft.so.2libpsl.so.5libXtst.so.6libboost_thread.so.1.62.0libm.so.6libatk-bridge-2.0.so.0libxcb-icccm.so.4libgs.so.9libbluetooth.so.3libhistory.so.7libformw.so.5libBrokenLocale.so.1libopencv_video.so.2.4libgsttag-1.0.so.0libXau.so.6libbabeltrace-ctf-metadata.so.1libopenmpt_modplug.so.1libfltk.so.1.3libwbclient.so.0libnl-3.so.200libicui18n.so.57libfuse.so.2libgconf-2.so.4libGLEW.so.2.0libQt5DBus.so.5libgdata.so.22libharfbuzz.so.0libgraphite2.so.3liborc-0.4.so.0libncurses.so.5libgpg-error.so.0libwayland-client.so.0libwebrtc_audio_processing.so.1libcmis-c-0.5.so.5libmnl.so.0libicule.so.57libQt5Widgets.so.5libisc.so.160libuno_sal.so.3libgutenprint.so.2libClp.so.1libmtp.so.9libsoup-2.4.so.1liblirc_client.so.0libclucene-shared.so.1libply-splash-core.so.4libsrtp.so.0libX11.so.6libfdisk.so.1libopus.so.0libqpdf.so.17libIlmImf-2_2.so.22liblogging-stdlog.so.0libraw_r.so.15libgstbadvideo-1.0.so.0libspeexdsp.so.1libgnome-desktop-3.so.12libpigpioultrasonic.solibfontconfig.so.1libtheoraenc.so.1libidn2.so.0libicutu.so.57libserd-0.so.0libnss_mdns6_minimal.so.2libinput.so.10libflite_cmu_grapheme_lex.so.1libffi.so.6libtevent.so.0libssh-gcrypt_threads.so.4libexiv2.so.14libbabeltrace-ctf.so.1libxcb-xinerama.so.0libvisual-0.4.so.0libform.so.5libusb-0.1.so.4libassuan.so.0libXmuu.so.1libavahi-core.so.7libjpeg.so.62libopencv_imgproc.so.2.4libsmbclient.so.0libXaw.so.7libuv.so.1libwnck-3.so.0libwpg-0.3.so.3libsane.so.1libxshmfence.so.1libmenu-cache.so.3libdevmapper.so.1.02.1libsamba-errors.so.1libde265.so.0libeot.so.0libgstpbutils-0.10.so.0libpulse-simple.so.0libnss_mdns4.so.2libimobiledevice.so.6libselinux.so.1libbabeltrace-lttng-live.so.1libpng16.so.16libxcb-shape.so.0libdrm_freedreno.so.1libgstphotography-1.0.so.0libetpan.so.17libwiringPiDev.solibXrender.so.1libvorbisfile.so.3libflite_cmu_indic_lex.so.1libseccomp.so.2libboost_date_time.so.1.62.0libpciaccess.so.0libboost_iostreams.so.1.58.0libcupsfilters.so.1libaacs.so.0libxcb.so.1libboost_date_time.so.1.61.0libsidplay2.so.1libglade-2.0.so.0liblilv-0.so.0libelftoolchain.solibdaemon.so.0liborcus-parser-0.11.so.0libiw.so.30libwavpack.so.1libopenmaxil.solibndr-standard.so.0libgstwayland-1.0.so.0libijs-0.35.solibva-x11.so.1libgpm.so.2libwebp.so.6libjson-glib-1.0.so.0libass.so.5libXRes.so.1libpq.so.5libMagickWand-6.Q16.so.3libxapian.so.30libswresample.so.2libmpc.so.3libatspi.so.0libfftw3f.so.3libunique-1.0.so.0libXxf86vm.so.1libgdk-3.so.0libQt5PrintSupport.so.5libvo-amrwbenc.so.0liblockfile.so.1libsamba-hostconfig.so.0libIexMath-2_2.so.12libmad.so.0libruby-2.3.so.2.3libBLTlite.2.5.so.8.6libprocps.so.6libsystemd.so.0libkeybinder.so.0libjavascriptcoregtk-3.0.so.0libfftw3f_omp.so.3libpoppler-glib.so.8libCoinUtils.so.3libsemanage.so.1libgstriff-0.10.so.0libcidn.so.1ld-linux-armhf.so.3libetonyek-0.1.so.1libchromaprint.so.1libxcb-dri3.so.0libopencv_highgui.so.2.4libgdk-x11-2.0.so.0libfftw3.so.3librt.so.1libparted.so.2libgthread-2.0.so.0libcaca.so.0libXcomposite.so.1libgoa-1.0.so.0libgstbadbase-1.0.so.0libcupscgi.so.1libmjpegutils-2.1.so.0libgnutls.so.30libdca.so.0libdrm_radeon.so.1libavutil.so.55libpcre16.so.3libneon-gnutls.so.27libpord_seq-4.10.0.solibflite_cmu_us_kal16.so.1libgstreamer-0.10.so.0libfftw3_threads.so.3libssl.so.1.1libavformat.so.57libliveMedia.so.57libvlc.so.5libmp3lame.so.0libcddb.so.2libsbc.so.1libcmumps_seq-4.10.0.solibnss3.solibtxc_dxtn.solibpthread.so.0libnetapi.so.0libtheoradec.so.1libnetfilter_conntrack.so.3libgstbase-0.10.so.0libapparmor.so.1librtmp.so.1libclucene-core.so.1libQt5Xml.so.5libgstinterfaces-0.10.so.0libraw1394.so.11libbonobo-2.so.0libtcl8.6.solibcdio.so.13libcairo-gobject.so.2liblightdm-gobject-1.so.0libgirepository-1.0.so.1libImath-2_2.so.12libgphoto2_port.so.12libavcodec.so.57libbluray.so.2librom1394.so.0libdvdnav.so.4libqt5scintilla2.so.12libQt5EglFsKmsSupport.so.5libudev.so.0libgstbadaudio-1.0.so.0libdbus-1.so.3libsamba-policy.so.0libxkbfile.so.1libresid-builder.so.0libzmumps_seq-4.10.0.solibgcr-base-3.so.1libICE.so.6libedit.so.2liblzma.so.5libXcursor.so.1libsctp.so.1libgdbm.so.3libscsynth.so.1libpaper.so.1libva-drm.so.1libgstcodecparsers-1.0.so.0libattr.so.1libIlmThread-2_2.so.12libdw.so.1libyelp.so.0libwmflite-0.2.so.7libpopt.so.0libcrypt.so.1libxkbcommon.so.0libpng12.so.0libhunspell-1.4.so.0libtk8.6.solibxslt.so.1libkrb5.so.3libxklavier.so.16libgstnet-1.0.so.0libdconf.so.1libflite_cmu_grapheme_lang.so.1libwayland-egl.so.1libgtop-2.0.so.10liblua5.2-c++.so.0libelf.so.1liblouis.so.12libopencv_flann.so.2.4libvisio-0.1.so.1libcrypto.so.1.1libbabeltrace-ctf-text.so.1libply-splash-graphics.so.4libgif.so.7libicutest.so.57libflite_cmulex.so.1libndr-nbt.so.0libportaudio.so.2libx264.so.148libnss_mdns_minimal.so.2libntfs-3g.so.871libgroupsock.so.8libbz2.so.1.0libpostproc.so.54liblqr-1.so.0libpcreposix.so.3libOsiCommonTests.so.1libglapi.so.0libgfortran.so.3libexslt.so.0liba52-0.7.4.solibmspub-0.1.so.1libcurl-gnutls.so.4libcap-ng.so.0libstartup-notification-1.so.0libnfsidmap.so.0libkrb5support.so.0libgxps.so.2libieee1284.so.3libBasicUsageEnvironment.so.1libsmartcols.so.1libepoxy.so.0libperl.so.5.24libwebpdemux.so.2libsoup-gnome-2.4.so.1libXext.so.6libcupsppdc.so.1liblangtag.so.1libbcm_host.solibunistring.so.0libip4tc.so.0libgsttag-0.10.so.0libcmis-0.5.so.5libevdocument3.so.4libnotify.so.4libsqlite3.so.0libcolorhug.so.2libsensors.so.4libart_lgpl_2.so.2libXi.so.6libmpfr.so.4libsamplerate.so.0libext2fs.so.2libtirpc.so.1libQt5Sql.so.5libjavascriptcoregtk-4.0.so.18libgmodule-2.0.so.0libflite_cmu_us_slt.so.1libdvdread.so.4libgstreamer-1.0.so.0libv4l2.so.0libmpeg2.so.0libgstcheck-0.10.so.0libpangoxft-1.0.so.0: command not found
/opt/pihole/gravity.sh: line 29: $'\272\006\020\240\341\b\224\345k\237\377\353\220P\342[\272\f\224\345\b0\224\345S\341': command not found
/opt/pihole/gravity.sh: line 30: $'\006\020\240\341\210\237\377\353\220P\342S\272\001\020\240\343\f\224\345^\237\377\353\220P\342N\272\b\224\345\020\240\341': command not found
/opt/pihole/gravity.sh: line 31: $'S\343\006\001': command not found
/opt/pihole/gravity.sh: line 32: $'o\261\377\353\004\240\341\037\263\377\353\220\240\341\E\352\2649\324\341S\343\025\032\2301\224\345S\343\022\032\0301\224\345S\343\017\032\0341\224\345S\343\f\032\025\220\340\343\004\240\341\342\210\377\353\214': command not found
/opt/pihole/gravity.sh: line 33: $'\004\240\341\333J\377\353\220\240\341Y\343\037\272\0300\324\345': command not found
/opt/pihole/gravity.sh: line 34: $'\001\023\3431': command not found
/opt/pihole/gravity.sh: line 35: $'@\210\345\f\220\235\345\346\377\377\352\b\025\237\345\001\020\217\340~\377\377\3525\237\345\025\237\345\005\237\345\0030\217\340\374$\237\345\001\020\217\340\217\340\205\317\377\353\025\220\340\343\331\377\377\352\3504\237\345\350\024\237\345\350\004\237\345\0030\217\340\300': command not found
/opt/pihole/gravity.sh: line 36: $'\204\342\001\020\217\340\266\317\377\353\220P\342Y\272': command not found
/opt/pihole/gravity.sh: line 36: $'\0041\220\345\001': command not found
/opt/pihole/gravity.sh: line 37: $'\f0\325\345\200$\224\345\001\020\240\343\3400\003\342': command not found
/opt/pihole/gravity.sh: line 38: $'\300\322\345\b0\322\345': command not found
/opt/pihole/gravity.sh: line 39: ��3?�: No such file or directory
/opt/pihole/gravity.sh: line 29: $'\b\224\345P\343?\272\f0\224\345S\343\022\001\272\006\020\240\341\225\237\377\353\220P\342\205\342': command not found
/opt/pihole/gravity.sh: line 33: $'\001\240\343\032\313\377\353\320!\315\341': command not found
/opt/pihole/gravity.sh: line 48: unexpected EOF while looking for matching ``'
/opt/pihole/gravity.sh: line 110: syntax error: unexpected end of file
root@pi2014:~#
```
**Steps to reproduce:**

I repeated the installation script, and exactly the same happened.

**Debug token provided by [uploading `pihole -d` log](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug):**

I can't run pihole, I get this:

```
root@pi201
[gravity.sh.gz](https://github.com/pi-hole/pi-hole/files/2961781/gravity.sh.gz)

4:~# pihole -d log
bash: /usr/local/bin/pihole: cannot execute binary file: Exec format error
```

**Troubleshooting undertaken, and/or other relevant information:**
```
root@pi2014:~# cat /proc/cpuinfo
processor	: 0
model name	: ARMv6-compatible processor rev 7 (v6l)
BogoMIPS	: 697.95
Features	: half thumb fastmult vfp edsp java tls
CPU implementer	: 0x41
CPU architecture: 7
CPU variant	: 0x0
CPU part	: 0xb76
CPU revision	: 7

Hardware	: BCM2835
Revision	: 000e
Serial		: 00000000b8d27ada
```

**Comment by dschaper** [comment]
Can you run `head -n 30 /opt/pihole/gravity.sh` and `ls -lah /opt/pihole` please? The files look to be corrupted on your installation.

**Comment by mok0** [comment]
I flashed the SD card with a fresh raspian-lite version, did `apt update; apt upgrade; reboot` and re-ran the installer. The problem has now disappeared, the install script completed. I am sorry I cannot help to pin down the problem, since the old system is now gone. I installed the old system a couple of weeks ago and have been installing various stuff, but I didn't notice any problems with it until pi-hole failed.
